app/views/cookies/show.html.erb

```diff
@@ -1,3 +1,8 @@
+<%= render GovukComponent::BackLink.new(
+      text: 'Back',
+      href: @backlink,
+    )
+%>
```

Comment:
Please follow how this is done elsewhere, e.g.
`<% content_for :before_content, govuk_back_link(
  text: "Back",
  href: @backlink)
%>`

Reply:
It does leave a lot of space until the next element ?

Reply:
Please see this: https://design-system.service.gov.uk/components/back-link/#how-it-works:~:text=Always%20place%20back%20links%20at%20the,navigation%20links%2C%20including%20the%20back%20link